### PR TITLE
Attach photos to books

### PR DESCRIPTION
**Pull request by gillianpierce**
Pls review I think it is nice

### CODE REVIEW

**Comment by tymoorej** [comment]
Functionality works!

File: app/src/main/java/com/stl/skipthelibrary/HorizontalAdapter.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import android.view.ViewGroup;
+
+import java.util.List;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+public class HorizontalAdapter extends RecyclerView.Adapter<BookImageViewHolder> {
+
+    List<ViewableImage> bookImages;
```
Comment (LukeSlev) [inline app/src/main/java/com/stl/skipthelibrary/HorizontalAdapter.java]:
```suggestion
    private List<ViewableImage> bookImages;
```

File: app/src/main/java/com/stl/skipthelibrary/HorizontalAdapter.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    Context mContext;
```
Comment (LukeSlev) [inline app/src/main/java/com/stl/skipthelibrary/HorizontalAdapter.java]:
```suggestion
    private Context mContext;
```

File: app/src/main/java/com/stl/skipthelibrary/AddBooksActivity.java [hunk 3]
```diff
@@ -115,6 +159,27 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
+                ViewableImage newBookImage = new ViewableImage(getBitmapFromUri(imageUri));
+                bookImages.add(newBookImage);
+                horizontalAdapter.notifyDataSetChanged();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private Bitmap getBitmapFromUri(Uri uri) throws IOException {
```
Comment (LukeSlev) [inline app/src/main/java/com/stl/skipthelibrary/AddBooksActivity.java]: Maybe think about pulling this into viewable image and pass in context 🤔

File: app/src/main/java/com/stl/skipthelibrary/ViewableImage.java [hunk 4]
```diff
@@ -59,4 +59,6 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(getImageString());
     }
+
```
Comment (LukeSlev) [inline app/src/main/java/com/stl/skipthelibrary/ViewableImage.java]: Interesting addition here, not sure its what we want

Comment (gillianpierce) [reply app/src/main/java/com/stl/skipthelibrary/ViewableImage.java]: I'm not sure YOU are what we want

Comment (LukeSlev) [reply app/src/main/java/com/stl/skipthelibrary/ViewableImage.java]: 😢

**Pull request merged**